src/js/utils/ActionDots.jsx

```diff
@@ -104,7 +119,6 @@ ActionDots.propTypes = {
     defaultLabel: PropTypes.string.isRequired,
     href: PropTypes.string,
     reactLink: PropTypes.bool,
-    appendId: PropTypes.bool,
```

Comment:
to clean everything out I think we could also remove the `appendId` from the places where you were sending it
e.g:
https://github.com/openboxes/openboxes/blob/9d783dcc5d668d05f244dd9036f6591241ce21f8/src/js/components/stock-movement/inbound/StockMovementInboundTable.jsx#L77